components/Nav/Nav.js

```diff
@@ -86,7 +86,7 @@ function Nav(props = {}) {
               />
             </Menu.Link>
           </Box>
-          <Box as="li">
+          {/* <Box as="li">
```

Comment:
Would it be better to remove it, instead of commenting out?

Reply:
Yeah you're probably right lol, I was gonna leave it just in case someone changed their mind down the road. But let's just delete for now to clean up the code 👍